server/src/main/java/org/opensearch/wlm/cancellation/ShortestRunningTaskFirstSelectionStrategy.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.wlm.cancellation;
+
+import org.opensearch.tasks.Task;
+
+import java.util.Comparator;
+
+/**
+ * Represents a task selection strategy that prioritizes the shortest running tasks first.
+ */
+public class ShortestRunningTaskFirstSelectionStrategy extends AbstractTaskSelectionStrategy {
+
+    /**
+     * Returns a comparator that sorts tasks based on their start time in ascending order.
+     *
+     * @return The comparator
+     */
+    @Override
+    public Comparator<Task> sortingCondition() {
+        return Comparator.comparingLong(Task::getStartTime).reversed();
+    }
```

Comment:
Unused class? Can we remove this?

Reply:
+1

Reply:
I wanted to give these as choices to the customer and they could configure this with a setting. 
Though the setting changes are not in this PR.